### PR TITLE
feat(android/app): Add language picker for keyboard package installation

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses most of #3476 as listed below (rest of issue will be deferred to separate PR):
* Use a bottom nav bar for "BACK" and "NEXT" actions ("NEXT" can also be used for "INSTALL" or "OK")
* For installing a keyboard package, allow the user to choose several languages (based on kmp.json)
* SelectLanguageActivity changed to use a new layout `activity_select_language` and displays a list of language names sorted alphabetically. On new package installs, the bottom nav bar is displayed.

### readme
new readme

### Language picker
new select

### Welcome
new welcome

### Final screen (after hitting "OK" on welcome)
Since PackageActivity finishes, the "last" Settings screen is what is shown
screen final

------
I've included screens of lexical model package for comparison

### readme
new lexical model

### welcome (package doesn't include a file)
new lexical welcome

-------------------
For a keyboard package with only 1 languge (e.g. khmer_angkor), the NEXT button appears as "INSTALL". Clicking it skips the language picker and goes to install the package, and finally display welcome.htm.

Screenshot of the readme.htm page with the "INSTALL" button
new khmer

## TODO (for future PR)
- [ ] Implement progress dots of the Stepper

### CODE REVIEW

**Comment by darcywong00** [comment]
Re: long keyboard package names, the string gets truncated in this example
welcome long package name

File: android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java [hunk 1]
```diff
@@ -1093,7 +1093,7 @@ public void onPackageInstalled(List<Map<String, String>> keyboardsInstalled) {
         hashMap.get(KMManager.KMKey_LanguageID),
         hashMap.get(KMManager.KMKey_LanguageName),
         hashMap.get(KMManager.KMKey_Version),
-        hashMap.get(KMManager.KMKey_HelpLink),
+        hashMap.get(KMManager.KMKey_CustomHelpLink),
```
Comment (darcywong00) [inline android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java:1096]: This is a separate fix for displaying offline help. There used to be a distinction between 
KMKey_HelpLink - link to the keyboard page on help.keyman.com/keyboard/ 
KMKey_CustomHelpLink - offline welcome.htm

But now KMKey_HelpLink isn't really used (Keyboard class will generate the help link if welcome.htm doesn't exist)

**Comment by mcdurdin** [comment]
A few notes:

* I think the `Next` and `Install` buttons need to be in the top banner, not underneath, viz:

image

* Then it looks like the example in https://github.com/keymanapp/keyman/pull/3481#issuecomment-673358617 is wrapped, not truncated?

* Also the font weight and size for the title varies from screen.

I have not yet reviewed the code.

**Comment by darcywong00** [comment]
> * Then it looks like the example in [#3481 (comment)](https://github.com/keymanapp/keyman/pull/3481#issuecomment-673358617) is wrapped, not truncated?

It's wrapped, but the version string that should appear at the end is truncated

**Comment by darcywong00** [comment]
PR going back to wip to move the top right button to the title bar

**Comment by darcywong00** [comment]
ok, I think this is ready to review again. I've modified the PR description with current screenshots.

**Review by mcdurdin — COMMENTED** [review]
Looks very good. A couple of minor comments only, and:

* The `NEXT ->` button on the language picker page should be `INSTALL`

**Pull request merged**